src/food/food.controller.ts

```diff
@@ -24,102 +24,111 @@ export class FoodController {
   }
 
   @Get()
-  @ApiOperation({
-    summary: '음식 리스트를 가져오는 API',
-    description: '사용자 id, category가 주어질 경우 그에 맞는 음식만 가져온다.',
-  })
-  @ApiQuery({
-    name: 'userId, category, hotLevel, sort, size',
-    description: 'userId, category, hotLevel, sort, size',
-    type: FindFoodsQueryDto,
-  })
-  @ApiResponse({ description: '음식 리스트', type: FindFoodDto })
+  @ApiOperation({ summary: '음식 리스트를 가져오는 API' })
   async findFoods(@Query() params: FindFoodsQueryDto): Promise<FindFoodDto[]> {
     return this.foodService.findFoods(params);
   }
 
   @Get('/reviews')
-  @ApiOperation({
-    summary: '리뷰할 음식 리스트를 가져오는 API',
-    description: '리뷰 할 음식 List을 가져온다.',
-  })
-  @ApiCreatedResponse({ description: '음식 list', type: Food })
+  @ApiOperation({ summary: '리뷰할 음식 리스트를 가져오는 API' })
   async reviewFoodList(): Promise<Food> {
-    return this.foodService.findReviewFoods();
+    try {
+      const food: Promise<Food> = this.foodService.findReviewFoods();
+      return food;
+    } catch (e) {
+      throw new HttpException(ERROR_MESSAGE.NOT_FOUND, HttpStatus.BAD_REQUEST);
+    }
   }
 
   @Get('/tests')
-  @ApiOperation({
-    summary: '테스트 음식 리스트를 가져오는 API',
-    description: '테스트에 필요한 음식 리스트를 가져온다.',
-  })
-  @ApiCreatedResponse({ description: '음식 list', type: Food })
+  @ApiOperation({ summary: '테스트 음식 리스트를 가져오는 API' })
   async findTestFoodList(): Promise<Food[]> {
-    return this.foodService.findTestFoods();
+    try {
+      const foods: Promise<Food[]> = this.foodService.findTestFoods();
+      return foods;
```

Comment:
```suggestion
      return this.foodService.findTestFoods();
```

await을 해서 return하는 것이 아니기 때문에 굳이 변수에 할당할 필요가 없어 보입니다.